docs/app/pages/components/sections/tables/filters/filters.component.html

```diff
@@ -0,0 +1,246 @@
+                or <code>FilterRemoveAllEvent</code> and will contain the filter being added or removed in a filter property 
+                (remove all will no contain affected filters).</td>
+        </tr>
+
+    </tbody>
+</table>
+
+<h4>Filter Dropdown</h4>
+
+<p>The dropdown filter (shown as Status in the demo above) can be added inside the filter container using the 
+    <code>ux-filter-dropdown</code> component.</p>
+
+<p>The following attributes can be used to customise the behavior of the component:</p>
+
+<table class="table">
+
+    <thead>
+        <tr>
+            <th>Name</th>
+            <th>Type</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+
+    <tbody>
+
+        <tr>
+            <td><b>@Input</b> filters</td>
+            <td><b>Filter[]</b></td>
+            <td>Sets the list of filters to be displayed.</td>
+        </tr>
+
+        <tr>
+            <td><b>@Input</b> initial</td>
+            <td><b>Filter</b></td>
+            <td>Sets the initial filter state, this should be set to the state where nothing has been filtered as it will revert to this when the filters have been removed.</td>
+        </tr>
+
+    </tbody>
+</table>
+
+<h4>Filter Options</h4>
+
+<p>A filter object can contain the follwing properties:</p>
+
+<table class="table">
+
+    <thead>
+        <tr>
+            <th>Name</th>
+            <th>Type</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+
+    <tbody>
+
+        <tr>
+            <td>group</td>
+            <td><b>string</b></td>
+            <td>Sets the filter group, all filters in the same filter dropdown should have the same group.</td>
+        </tr>
+
+        <tr>
+            <td>title</td>
+            <td><b>string</b></td>
+            <td>The text which will be displayed on the button when this filter is active.</td>
+        </tr>
+
+        <tr>
+            <td>name</td>
+            <td><b>string</b></td>
+            <td>The text which will be displayed in the dropdown menu.</td>
+        </tr>
+
+        <tr>
+            <td>initial</td>
+            <td><b>boolean?</b></td>
+            <td>One of the filters in a group should be set to initial, this refers to the unfiltered state.</td>
+        </tr>
+
+    </tbody>
+</table>
+
+<h3>Filter Dynamic</h3>
+
+<p>When there are a large number of filter options, the Dynamic Filter can be used to replace the regular 
+    dropdown with a text search, supporting dropdown typeahead. This is configurable to fallback to the 
+    standard implementation when the list of filters falls beneath a certain length. An example of this is shown 
+    as Author in the demo above. 
+</p>
+
+<p>The dynamic filter can be added inside the filter container using the <code>ux-filter-dynamic</code> component.</p>
+
+<p>The following attributes fcan be used to customise the behavior of the component:</p>
```

Comment:
Would be good to add a section here about Custom Filters and how teams can create their own filters - similar to the custom facets section.